samples/bluetooth/mesh/README.rst

```diff
@@ -44,9 +44,13 @@ For other boards, build and flash the application as follows:
 Refer to your :ref:`board's documentation <boards>` for alternative
 flash instructions if your board doesn't support the ``flash`` target.
 
-To run the application on an :ref:`nrf5340dk_nrf5340`, a Bluetooth controller application
-must also run on the network core. The :ref:`bluetooth-hci-ipc-sample` sample
-application may be used. Build this sample with configuration
+Additional kconfig options need to be set on the Bluetooth controller
```

Comment:
It is worth to set this off as a subsection in sample docs (and move it as a last para):
```suggestion
Bluetooth controller on different device:
=========================================
Additional kconfig options need to be set on the Bluetooth controller
```